packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -70,6 +74,7 @@ const propTypes = {
 const defaultProps = {
   layout: Layouts.ONE_COLUMN,
   textEmphasis: TextEmphasisTypes.DEFAULT,
+  overrideDefaultStyling: false,
   isTruncated: false,
```

Comment:
Just thinking aloud so please correct me if I am wrong. Do we have any potential conflicts between the overrideDefaultStyling and other props(i.e. isTruncated). For example, if the consumer uses their original styling and it is a truncated text, but they keep the isTruncated as false(by default) at the same time. Will this scenario cause any issues?

Reply:
No it shouldn't cause issues, overrideDefaultStyling just effects the text styling (color, size, bolding, etc) so truncation should not be effected.

Reply:
Ok so I added some temp examples locally to test some stuff out. Truncated and overflow are working well with the override.

Single column truncation
<img width="468" alt="Screenshot 2023-08-16 at 8 50 11 AM" src="https://github.com/cerner/terra-clinical/assets/40574651/3273916c-d95b-4559-ad9d-69751ecb93a4">

Two column truncation
<img width="920" alt="Screenshot 2023-08-16 at 8 50 39 AM" src="https://github.com/cerner/terra-clinical/assets/40574651/af56ca2c-022e-4893-b76f-aa39578c2569">

Overflow
<img width="920" alt="Screenshot 2023-08-16 at 8 53 04 AM" src="https://github.com/cerner/terra-clinical/assets/40574651/75f4e521-7ac4-47b8-9170-dcd8f6b68b9e">


However, I tested out a display passed in with isTruncated prop as true while our itemview has isTruncated set to false - there were some formatting issues. Left is what it looks like when just itemview has isTruncated set to true, the right is when it's set to false (but the display has it set to true)
<img width="1553" alt="Screenshot 2023-08-16 at 8 59 27 AM" src="https://github.com/cerner/terra-clinical/assets/40574651/00baa3e5-55c8-48c4-90e8-526bbf3d607a">

This is an existing issue though, with or without the override.

Reply:
So this will likely need a story logged to fix this use case and for tests/examples to be added for the combo of isTruncated in display as true/isTruncated in itemview as false.

@scottwilmarth tagging you for visibility, should I go ahead and log a jira for the backlog?

Reply:
Anything related to the above comment will be handled in a different PR, we can just focus on the override logic in this story for now.